README.md

```diff
@@ -2,33 +2,76 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human {
+     constructor(name, age){
+       this.name = name;
+       this.age = age;
+     }
+
+     ageOneYear(str){
+       return this.age++
+     }
+
+      eating(str){
+        return "mmm, mmm, mmm, I'm love'n it."
+      }
+    }
+
```

Comment:
"Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it"."
Just a couple of notes for your eating function:
- no argument is needed
- your function returns the string, but we want to log it with console.log instead
- see comment for line 33!